lib/modules/manager/osgifeature/__fixtures__/double-slash-not-comment.json

```diff
@@ -0,0 +1,18 @@
+{
+    "bundles":[
+         {
+             "id":"com.h2database:h2-mvstore:2.1.214",
```

Comment:
Please also add some malfunction tests like missing `id` attributes inside of bundles.